fn generic_error() {
    print(ctx().mail_timestamp);
}

fn throw_error() {
    throw "this is an error throwned by the user";
}
